app/extensions/torrent/webtorrent.html

```diff
@@ -10,10 +10,10 @@
     <link rel="shortcut icon" type="image/x-icon" href="img/webtorrent-128.png">
     <title data-l10n-id="webtorrentPage"></title>
     <script src="ext/l20n.min.js"></script>
-    <script src='gen/webtorrentPage.entry.js'></script>
     <link rel="localization" href="locales/{locale}/app.properties">
   </head>
   <body>
     <div id="appContainer" />
+    <script src='gen/webtorrentPage.entry.js'></script>
```

Comment:
This let's us use the DOM node `#appContainer` right away, without waiting for DOM ready.